sunpy/timeseries/metadata.py

```diff
@@ -78,8 +86,10 @@ class TimeSeriesMetaData:
     def __init__(self, meta=None, timerange=None, colnames=None):
         self.metadata = []
         # Parse in arguments
-        if not isinstance(meta, type(None)):
-            if isinstance(meta, (dict, MetaDict)) and isinstance(timerange, TimeRange) and isinstance(colnames, list):
+        if meta is not None:
+            if (isinstance(meta, (dict, MetaDict)) and
```

Comment:
Is there anyway to format this conditional a bit more nicely?

Reply:
The whole thing, or just this line? Maybe if `isinstance(MetaDict, dict) == True` then it could be `isinstance(meta, dict)`.

Reply:
Sorry, I meant just the multi-line `if` statement starting on line 90

Reply:
Agree it's a bit unweildy, but not sure how it can be made simpler.